package.json

```diff
@@ -53,11 +53,11 @@
     "format:js": "prettier --write '**/*.js'",
     "format:md": "prettier --write --parser markdown '**/*.md'",
     "format:yaml": "prettier --write --parser yaml '**/*.yaml'",
-    "test": "jest --config jest.test.config.js",
-    "test:watch": "jest --config jest.test.config.js --watch",
-    "test:bundle": "jest --config jest.bundle.config.js",
+    "test": "cross-env NODE_ICU_DATA=node_modules/full-icu jest --config jest.test.config.js",
```

Comment:
From here

https://github.com/formatjs/react-intl/issues/1179#issuecomment-428183019

Reply:
Otherwise tests that rely on locale stuff will fail

Reply:
In appkit I did it like this:

https://github.com/commercetools/merchant-center-application-kit/blob/b6200086597c97640852052618cedc7cdb17f3b5/packages/jest-preset-mc-app/setup-test-framework.js#L26-L31

Reply:
I tried this first 😢 

I think it's a different issue.

Reply:
I don't think installing the full icu is necessary. Patching the Intl in the jest setup fixed the problem for me.

What is the problem you're having?

Reply:
Turns out just needed more polyfills 😓